lib/Data/Chronicle.pm

```diff
@@ -10,7 +10,7 @@ Version 0.07
 
 =cut
 
-our $VERSION = '0.07';
+our $VERSION = '0.08';
```

Comment:
We're still discussing this, so things may change at some point, but:

For now I'd recommend not updating the version during development. We'll update the version during the merge to master, this should help to avoid merge conflicts and ensure that we have consistent version history in the master branch.